components/DeleteButton.tsx

```diff
@@ -13,12 +12,16 @@ export default function DeleteButton({ itemId, title }: DeleteButtonProps) {
     throw new Error('item is undefined');
   }
 
-  const router = useRouter();
+  const handleDelete = async () => {
+    try {
+      await deleteItems(itemId);
+      window.location.reload();
```

Comment:
I'm a little stumped on this one so might need to call in a second opinion 🤔 @nichgalzin 

The advantage of useRouter is that it prevents a full page reload and helps us preserve state, so although this option here seems to be working I worry about it causing some small bug in our state management that we're not catching right now.

That being said, I wasn't able to understand so far why router.refresh isn't doing it 🙃